src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,38 +2,61 @@
 
 public class DefaultCountingOutRhymer {
 
+    private static final int TOTAL_FULL_VALUE = 11;

```

Comment:
This constant is related to MAX_SIZE . Changing only one leads to an error (inconsistency).